docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,240 @@ sidebar_label: Wikidata
+
+You can export any schema you create, and import an existing schema for use with a new dataset. This can help you work in batches on a large amount of data with a minimum of redundant labor.
+
+Once you select <span class="menuItems">Edit Wikidata schema</span> under the <span class="menuItems">Extensions</span> dropdown menu, your project interface will change. You’ll see new tabs added to the right of “X rows/records" in the grid header: “Schema,” “Issues,” and “Preview.” You can now flip between the tabular grid format of your dataset, or the screens that allow you to prepare data for uploading. 
+
+OpenRefine presents you with an easy visual way to map out the relationships in your dataset. Each of the columns of your project will appear at the top of the sceren, and you can simply drag and drop them into the appropriate slots. To get start, select one column as an item. 
+
+![A screenshot of the schema construction window in OpenRefine.](/img/wikidata-schema.png)
+
+There is [a Wikidata tutorial on how OpenRefine handles Wikidata schema](https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine/Editing/Tutorials/Basic_editing).
+
+#### Editing terms with your schema
+
+You may wish to include edits to [terms](#terms-labels-descriptions-aliases-and-sitelinks) (labels, aliases, descriptions, or sitelinks) as well as establishing relationships between entities. 
+
+For example, you may wish to upload pseudonyms, pen names, maiden or married names for historical authors. You can do so by putting the preferred names in one column of your dataset and alternative names in another column. In the schema interface, add an item for the preferred values, then click “Add term” on the right-hand side of the screen. Select “Alias” from the dropdown, enter in “English” in the language field, and drop your alternative names column into the space. 
+
+Terms must always have a language selected. You cannot edit multiple languages at once, unless you drop a suitable column into the “language” field. For example, if you had translated publication titles, with data in the format
+
+|English title|Translated title|Translation language|
+|---|---|---|
+|Possession|Besessen|German|
+||Обладать|Russian|
+|Disgrace|Disgrâce|French|
+||Vergogna|Italian|
+|Wolf Hall|En la corte del lobo|Spanish|
+||ウルフ・ホール|Japanese|
+
+You could upload translated titles to “Label” with the language from “Translation language.” You may wish to fetch the two-letter language code and use that instead for better language matches.
+
+### Manage Wikidata account
+
+To edit Wikidata directly from OpenRefine, you must have a Wikidata account and log in with it in OpenRefine. OpenRefine can only upload edits with Wikidata user accounts that are “[autoconfirmed](https://www.wikidata.org/wiki/Wikidata:Autoconfirmed_users)” - that is, accounts that have more than 50 edits and have existed for longer than four days. 
```

Comment:
I suspect these criteria are subject to change, so probably best to just point to wherever it's defined on Wikidata.

Reply:
Not since 2013. But I can add in "as of the time of writing."